src/main/scala/com/sky/kafka/topicloader/config/config.scala

```diff
@@ -0,0 +1,56 @@
+package com.sky.kafka.topicloader
+
+import java.time.Duration
+import java.util.concurrent.TimeUnit
+
+import cats.data.ValidatedNec
+import cats.implicits._
+import com.typesafe.config.{Config => TypesafeConfig, ConfigException}
+
+import scala.concurrent.duration.FiniteDuration
+import scala.util.{Failure, Success, Try}
+
+package object config {
```

Comment:
the case classes, objects and traits should be moved out of this package object: https://github.com/alexandru/scala-best-practices/blob/master/sections/2-language-rules.md#218-must-not-include-classes-traits-and-objects-inside-package-objects